src/app/components/leather-logo.tsx

```diff
@@ -11,16 +12,16 @@ export const LeatherLogo = memo((props: LeatherLogoProps) => {
   const { isClickable, onClick } = props;
 
   return (
-    <LeatherButton
+    <styled.button
```

Comment:
@mica000 @fabric-8 : We have some logic here so that the `Leather` logo is not clickable if you are on the home page.

What do you think about simplifying this and always letting users click it, even if on `Home`?

Reply:
Im cool with it

Reply:
Yeah I think that's fine